src/giskardpy/plugin_update_constraints.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, name, as_name):
         self.controlled_joints = set()
         self.controllable_links = set()
         self.last_urdf = None
+        self.allowed_types = [x for x in inspect.getmembers(giskardpy.constraints) if inspect.isclass(x[1])]
```

Comment:
- remove `allowed_constraint_names`
- turn the list of tuples into a dict, saves a few loops later on